src/net/sourceforge/zmanim/hebrewcalendar/JewishDate.java

```diff
@@ -643,7 +656,7 @@ private static void validateGregorianDate(int year, int month, int dayOfMonth) {
 	 *            {@link GregorianCalendar}, where {@link Calendar#JANUARY} has a value of 0.
 	 */
 	private static void validateGregorianMonth(int month) {
-		if (month > 11 || month < 0) {
+		if (month > Calendar.DECEMBER || month < Calendar.JANUARY) {

```

Comment:
Shouldn't this also use the MONTH_ constants?

Reply:
apparently not, according to its javadoc